miner/worker.go

```diff
@@ -83,6 +84,9 @@ const (
 var (
 	randomSeedString = []byte("Randomness seed string")
 	randomSeed       []byte
+
+	// Timer used to measure block finalization time from created to finalized
+	blockFinalizationTimer = metrics.NewRegisteredTimer("miner/block/finalization", nil)
```

Comment:
Looks like this needs to be changed to type gauge (e.g. https://github.com/celo-org/celo-blockchain/blob/master/consensus/istanbul/backend/backend.go#L123).